### DISCUSSION TITLE
Adding certbot plugins

### DISCUSSION THREAD

**Discussion by toastwerks** [ZoeyVid/NPMplus#907] [Q&A] [answered]
I feel like I'm missing something obvious - is there a trick to adding a DNS plugin for certbot? Trying to set up DNS challenge and my provider isn't listed, but there is a plugin. I added it in the json file following the pattern, but it doesn't show up as an option in the dropdown.

**Comment by Zoey2936** [comment, marked as answer]
Hello, the changes will only apply if the frontend is rebuilt, please open a PR adding the plugin to the json file, I will merge it and release it, then you will be able to use it

**Reply by toastwerks** [reply]
Thank you! I'll work on getting that figured out.
